packages/core/src/app/cart/CartSummary.tsx

```diff
@@ -16,15 +16,26 @@ export type WithCheckoutCartSummaryProps = {
     storeCurrency: StoreCurrency;
     shopperCurrency: ShopperCurrency;
     storeCreditAmount?: number;
+    isNewMultiShippingUIEnabled: boolean;
 } & RedeemableProps;
 
-const CartSummary: FunctionComponent<WithCheckoutCartSummaryProps> = ({ cartUrl, ...props }) => {
-    const headerLink = isBuyNowCart() ? null : <EditLink url={cartUrl} />;
+const CartSummary: FunctionComponent<
+    WithCheckoutCartSummaryProps & {
+        isMultiShippingMode: boolean;
+    }
+> = ({ cartUrl, isMultiShippingMode, isNewMultiShippingUIEnabled, ...props }) => {
+    const headerLink = isBuyNowCart() ? null : (
+        <EditLink
+            isMultiShippingMode={isNewMultiShippingUIEnabled && isMultiShippingMode}
```

Comment:
I see this will be rolled out for all users regardless of the experiment, is that expected?

Reply:
Saw this https://github.com/bigcommerce/checkout-js/pull/2055/files#diff-ea06ebdcd56f95f158f566d4d907e9e54c508436fac6c5e65900160b0aedd7f7R28.